Add doc for alerting comments (#7360)
opensearch-project/documentation-website: _observing-your-data/alerting/settings.md

`plugins.scheduled_jobs.sweeper.backoff_millis` | 50ms | The amount of time the sweeper waits between retries---increases exponentially after each failed retry.
`plugins.scheduled_jobs.sweeper.retry_count` | 3 | The total number of times the sweeper should retry before throwing an error.
`plugins.scheduled_jobs.request_timeout` | 10s | The timeout for the request that sweeps shards for jobs.

> `plugins.alerting.comments_enabled` | "false" | Whether comments for the Alerting plugin is enabled or not.

Reviewer: ```suggestion
`plugins.alerting.comments_enabled` | `false` | Enables or disables comments for the Alerting plugin.
```

`plugins.alerting.comments_history_max_docs` | 1000 | The maximum number of comments to store in the `.opensearch-alerting-comments-history-<date>` index before creating a new index.

> `plugins.alerting.comments_history_max_age` | 30d | The oldest document to store in a `.opensearch-alerting-comments-history-<date>` index before creating a new one. If the number of comments in this time period does not exceed `comments_history_max_docs`, one index is created per period (for example, one every 30 days).

Reviewer 2: ```suggestion
`plugins.alerting.comments_history_max_age` | 30d | The oldest document to store in an `.opensearch-alerting-comments-history-<date>` index before creating a new one. If the number of comments in the specified time period does not exceed `comments_history_max_docs`, then 1 index is created per period (for example, 1 every 30 days).
```

> `plugins.alerting.comments_history_max_age` | 30d | The oldest document to store in the `.opensearch-alerting-comments-history-<date>` index before creating a new index. If the number of comments in this time period does not exceed comments_history_max_docs, the Alerting plugin creates one comments history index per period (e.g. one index every 30 days).

Reviewer: ```suggestion
`plugins.alerting.comments_history_max_age` | 30d | The oldest document to store in a `.opensearch-alerting-comments-history-<date>` index before creating a new one. If the number of comments in this time period does not exceed `comments_history_max_docs`, one index is created per period (for example, one every 30 days).
```

> `plugins.alerting.comments_history_rollover_period` | 12h | How often to check if the `.opensearch-alerting-comments-history-write` alias should roll over to a new index and delete old comment history indexes.

Reviewer 2: ```suggestion
`plugins.alerting.comments_history_rollover_period` | 12h | How often to determine whether the `.opensearch-alerting-comments-history-write` alias should roll over to a new index and delete old comment history indexes.
```

> `plugins.alerting.comments_history_rollover_period` | 12h | How frequently to check whether the .opensearch-alerting-comments-history-write alias should roll over to a new history index and whether the Alerting plugin should delete any comments history indexes.

Reviewer: ```suggestion
`plugins.alerting.comments_history_rollover_period` | 12h | How often to check if the `.opensearch-alerting-comments-history-write` alias should roll over to a new index and delete old comment history indexes.
```

> `plugins.alerting.comments_history_retention_period` | 60d | The time to keep comment history indexes before automatic deletion.

Reviewer 2: ```suggestion
`plugins.alerting.comments_history_retention_period` | 60d | The amount of time to keep comment history indexes before automatic deletion.
```

> `plugins.alerting.comments_history_retention_period` | 60d | The time to keep comment history indexes before automatic deletion.

Reviewer 2: "keep" => "store"?

> `plugins.alerting.comments_history_retention_period` | 60d | The amount of time to keep comments history indexes before automatically deleting them.

Reviewer: ```suggestion
`plugins.alerting.comments_history_retention_period` | 60d | The time to keep comment history indexes before automatic deletion.
```

`plugins.alerting.max_comment_character_length` | 2000 | The maximum character length of a comment.
`plugins.alerting.max_comments_per_alert` | 500 | The maximum number of comments that can be posted on an alert.

> `plugins.alerting.max_comments_per_notification` | 3 | The maximum number of comments per alert to include in the `ctx` mustache template variable for alert notifications.

Reviewer 2: ```suggestion
`plugins.alerting.max_comments_per_notification` | 3 | The maximum number of comments per alert to include in the `ctx` Mustache template variable for alert notifications.
```
